app/services/LegacyIdentityDbUserService.scala

```diff
@@ -0,0 +1,33 @@
+package services
+
+import model.User
+import slick.basic.DatabaseConfig
+import slick.jdbc.JdbcProfile
+import slick.lifted.TableQuery
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class LegacyIdentityDbUserService(val dbConfig: DatabaseConfig[JdbcProfile])(implicit ec: ExecutionContext)
+    extends UserService {
+
+  import dbConfig.profile.api.*
+
+  private val db = dbConfig.db
+
+  private class UserTable(tag: Tag) extends Table[User](tag, "users") {
+    def identityId = column[String]("id")
+    def brazeId = column[String]("braze_uuid")
+    def * = (identityId, brazeId).mapTo[User]
+  }
+
+  private val users = TableQuery[UserTable]
+
+  override def healthCheck(): Future[Unit] =
+    db.run(sql"SELECT 1".as[Int]).map(_ => ())
+
+  def fetchUserByIdentityId(id: String): Future[Option[User]] = {
+    val action = users.filter(_.identityId === id).take(1).result
```

Comment:
what is the advantage of using slick ? it seems like it makes it harder to see the actual sql that is running

Reply:
Not much unless the database rows correspond very closely with case classes.  So can stick with Slick or use Anorm instead

Reply:
I don't know if it's going to be worth it considering most of the data is going to be in a json field that probably is not well supported.

We can give it a try given that you already wrote this code, but in my experience it gets a bit confusing when you delegate the sql generation to a library.

Reply:
All right.  Stick with it for now.  I don't think it will make much difference - it's going to be a bit messy either way